main/src/main/java/org/mobilitydata/gtfsvalidator/validator/RouteNameValidator.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.RouteBothShortAndLongNameMissingNotice;
+import org.mobilitydata.gtfsvalidator.notice.RouteShortAndLongNameEqualNotice;
+import org.mobilitydata.gtfsvalidator.notice.RouteShortNameTooLongNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsRoute;
+
+/**
+ * Validates short and long name for a single route.
+ * <p>
+ * Generated notices:
+ * * RouteBothShortAndLongNameMissingNotice
+ * * RouteShortAndLongNameEqualNotice
+ * * RouteShortNameTooLongNotice
+ */
+@GtfsValidator
+public class RouteNameValidator extends SingleEntityValidator<GtfsRoute> {
+    private static final int MAX_SHORT_NAME_LENGTH = 6;
```

Comment:
Spec best practices say max of 12 characters:
https://gtfs.org/best-practices/#routestxt

Reply:
Indeed! And Google uses 6 characters internally. OK, let's use the public best practices.